dotnet/targets/dotnet-xcsync.targets

```diff
@@ -0,0 +1,72 @@
+<Project>
+	<Target Name="ComputeXcsyncArguments">
+		<PropertyGroup>
+			<XcsyncExecute>$(DOTNET_HOST_PATH) xcsync</XcsyncExecute>
+			<XcsyncProjectFile Condition="'$(XcsyncProjectFile)' == ''">$(MSBuildProjectFullPath)</XcsyncProjectFile>
+			<XcsyncXcodeFolder Condition="'$(XcsyncXcodeFolder)' == ''">$(IntermediateOutputPath)xcode</XcsyncXcodeFolder>
+			<XcsyncTargetFrameworkMoniker Condition="'$(XcsyncTargetFrameworkMoniker)' == ''">$(TargetFramework)</XcsyncTargetFrameworkMoniker>
+			<XcsyncVerbosity Condition="'$(XcsyncVerbosity)' == ''">Diagnostic</XcsyncVerbosity>
+			<XcsyncOpenXcode Condition="'$(XcsyncOpenXcode)' == ''">true</XcsyncOpenXcode>
+
+			<XcsyncCommonArguments Condition="'$(XcsyncProjectFile)' != ''">$(XcsyncCommonArguments) -p "$(XcsyncProjectFile)"</XcsyncCommonArguments>
+			<XcsyncCommonArguments Condition="'$(XcsyncXcodeFolder)' != ''">$(XcsyncCommonArguments) -t "$(XcsyncXcodeFolder)"</XcsyncCommonArguments>
+			<XcsyncCommonArguments Condition="'$(XcsyncTargetFrameworkMoniker)' != ''">$(XcsyncCommonArguments) -tfm $(XcsyncTargetFrameworkMoniker)</XcsyncCommonArguments>
+			<XcsyncCommonArguments Condition="'$(XcsyncVerbosity)' != ''">$(XcsyncCommonArguments) -v $(XcsyncVerbosity)</XcsyncCommonArguments>
+			<XcsyncCommonArguments>$(XcsyncCommonArguments) --dotnet-path "$([System.IO.Path]::GetDirectoryName($(DOTNET_HOST_PATH)))"</XcsyncCommonArguments>
+
+			<XcsyncGenerateArguments>$(XcsyncCommonArguments) --force</XcsyncGenerateArguments>
+			<XcsyncGenerateArguments Condition="'$(XcsyncOpenXcode)' != 'false'">$(XcsyncGenerateArguments) --open</XcsyncGenerateArguments>
+
+			<XcsyncSyncArguments>$(XcsyncCommonArguments)</XcsyncSyncArguments>
+		</PropertyGroup>
+	</Target>
+
+	<Target Name="CheckXcsyncInstalled">		
+		<PropertyGroup>
+			<LocalErrorCode>0</LocalErrorCode>
+			<GlobalErrorCode>0</GlobalErrorCode>
+			<XcsyncInstalled>False</XcsyncInstalled>	
+		</PropertyGroup>
+	    <Exec ContinueOnError="True" IgnoreStandardErrorWarningFormat="true" Command="$(XcsyncExecute)" StandardErrorImportance="Low" StandardOutputImportance="Low">
+       		<Output TaskParameter="ExitCode" PropertyName="GlobalErrorCode"/>
+    	</Exec>
+	    <Exec ContinueOnError="True" IgnoreStandardErrorWarningFormat="true" Command="$(DOTNET_HOST_PATH) tool run xcsync" Condition="'$(GlobalErrorCode)' != '0'" StandardErrorImportance="Low" StandardOutputImportance="Low">
+       		<Output TaskParameter="ExitCode" PropertyName="LocalErrorCode"/>
+    	</Exec>
```

Comment:
Looks like there are some spaces instead of tabs here.

Reply:
Fixed.